lib/jitsi_meet.dart

```diff
@@ -229,13 +229,13 @@ class JitsiMeetingOptions
   String userEmail;
   String iosAppBarRGBAColor;
 
-  Map<FeatureFlagEnum, bool> featureFlags = new HashMap();
+  Map<FeatureFlagEnum, dynamic> featureFlags = new HashMap();
```

Comment:
I understand why we moved to dynamic, but the tighter type check is very nice. Is there some code design you can come up with that makes it hard for clients to put in the wrong type of value for a feature flag (other than having to read the docs)? Should we separate out the integer flags? Should we go away from an enum map and use a class to represent flags?

Reply:
I agree with @tuantvu ! A class could be a good alternative, considering that aspect.

Reply:
Sounds good to me, I will try to rewrite it.

Reply:
If you're short in time, I can try it, just tell me :wink:

Reply:
@TBG-FR 
Sure brother, I am bit occupied with PR for the main Jitsi Repo and some other stuff, if you wanna take stab at it go ahead :) . I can spend time on this over the weekend.

If you do start with the work, do drop a message, so we both don't end up working on the same thing.
More power to you brother :)

Reply:
I didn't had time yet ahah, any news on your side ?

Reply:
Nope :( .. don't think I would have time this week as well.